server/main.go

```diff
@@ -16,6 +17,14 @@ import (
 
 func init() {
 	dbConnection.ConnectToDatabase()
+

```

Comment:
Why have you added this? We already have a dedicated file for migration related code.

Reply:
in my case migration code didn't work so I added but now I got it the point
So, sorry for this

Reply:
> in my case migration code didn't work so I added but now I got it the point So, sorry for this

Ahh, no need for sorry or anything. If anything's not working on your side, then you can surely add some code but don't add that part while committing the changes. Also, never do `git add .` when you are pushing the changes. Always add files one by one.